### PR TITLE
Autocode.com | New Presence

### PR DESCRIPTION
**Pull request by CTK-WARRIOR**
**Website** : https://autocode.com

**About** : Autocode is an API-native re-imagining of the software delivery process.
It's a web editor, runtime for apps, and hosting platform that treats the API as the foundational primitive of web development. It comes packaged with its very own Standard Library that anybody can contribute to.

**Screen Shots**

### CODE REVIEW

File: websites/A/Autocode/presence.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const presence = new Presence({
+clientId: "858292108195921920"
+}), supportedLanguages: Array<string> = [ "js", "md", "json", "gitignore", "txt", "html", "css"];
```
Comment (SlowLife1661) [inline websites/A/Autocode/presence.ts:3]: You don't really have to give it a type here, just a comment no change is required

**Review by Slowlife01 — APPROVED** [review]

File: websites/A/Autocode/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "altnames": ["standard library", "stdlib", "acode"],
+  "description": {
+    "en": "New user notifications. Purchase alerts. Scheduled jobs. Community bots. SQL queries. Internal tooling. Create whatever you want with JavaScript and APIs."
+  },
+  "url": "autocode.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/KNLjAwu.jpg",
+  "thumbnail": "https://i.imgur.com/iojaJvv.jpg",
+  "color": "#ffffff",
+  "tags": ["code", "IDE", "editor"],
```
Comment (SlowLife1661) [inline websites/A/Autocode/dist/metadata.json:17]:
```suggestion
  "tags": ["code", "ide", "editor"],
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — DISMISSED** [review]

File: websites/A/Autocode/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "CTK WARRIOR",
+    "id": "450634297871695896"
+  },
+  "service": "Autocode",
+  "altnames": ["standard library", "stdlib", "acode"],
```
Comment (Bas950) [inline websites/A/Autocode/dist/metadata.json:8]:
```suggestion
```

Comment (Bas950) [reply websites/A/Autocode/dist/metadata.json:8]: you can add those as tags

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**